sklearn/metrics/pairwise.py

```diff
@@ -341,7 +341,7 @@ def euclidean_distances(
 
     Notes
     -----
-    To achieve a better accuracy, `X_norm_squared` and `Y_norm_squared` may be
+    To achieve a better accuracy, `X_norm_squared` and `Y_norm_squared` may be
```

Comment:
I can not tell what this change is. A unicode character?

Reply:
There still is a single commit per rule, so there's the workaround of reviewing the commits individually.

In this case, I manually changed a NO-BREAK SPACE U+00A0 into a regular SPACE U+0020 in https://github.com/scikit-learn/scikit-learn/pull/30694/commits/0aec43b51a914e237683a09e0a569d9e967cb903, to silence [ambiguous-unicode-character-docstring (RUF002)](https://docs.astral.sh/ruff/rules/ambiguous-unicode-character-docstring/#ambiguous-unicode-character-docstring-ruf002).